db/migrations/0003_add_frames_table.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE frames (
```

Comment:
Sounds good. Why does it need to be hashes, can't we just store the uuid of the previous frame?